csrc/aio/py_test/parse_aio_stats.py

```diff
@@ -107,12 +85,7 @@ def get_metric(file, metric):
     with open(file) as f:
         for line in f.readlines():
             if line.startswith(METRIC_SEARCH[metric]):
-                if metric == RAW_RATE:
-                    fields = line.split()
-                    raw_time_sec = float(fields[2]) / 1e06
-                    raw_rate = (thread_count * num_giga_bytes * 1.0) / raw_time_sec
-                    return raw_rate
-                elif metric in [READ_SPEED, WRITE_SPEED]:
+                if metric in [READ_SPEED, WRITE_SPEED]:

```

Comment:
should the `PERF_METRICS` list created above be used instead? Or maybe there are more possible perf metrics that shouldn't trigger this branch.

Reply:
Yes, there could perf metrics we want to ignore or handle differently.